src/main/java/ch/njol/skript/expressions/ExprCommand.java

```diff
@@ -44,10 +44,10 @@
 @Description("The command that caused an 'on command' event (excluding the leading slash and all arguments)")
 @Examples({"# prevent any commands except for the /exit command during some game",
 		"on command:",
-		"{game.%player%.is playing} is true",
-		"command is not \"exit\"",
-		"message \"You're not allowed to use commands during the game\"",
-		"cancel the event"})
+		"\tif {game::%player%::playing} is true:",

```

Comment:
Using nested if statements is a bad practise, unless you actually need to use them.

Reply:
Inline conditions don't even look like a condition